lms/static/scripts/frontend_apps/config.ts

```diff
@@ -205,28 +205,40 @@ export type Product = {
  * the backend code.
  */
 export type ConfigObject = {
+  // Configuration present in all modes.
   mode: AppMode;
   api: {
     authToken: string;
-    sync: APICallInfo;
-    viaUrl: APICallInfo;
+
+    // Only present in "basic-lti-launch" mode.
+    sync?: APICallInfo;
+    viaUrl?: APICallInfo;
   };
+  dev: boolean;
```

Comment:
Is `dev` always present?

Reply:
Yes. It is among the global properties set here: https://github.com/hypothesis/lms/blob/0edc7c52da3aecc939fb7d433cb92ca44c3bc474/lms/resources/_js_config/__init__.py#L401.